config/checkstyle/space-clean

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+git status | grep java | awk '{print $3}' | xargs -o -n 1 vim -c ":retab" -c ":%s/\s\+$//g" -c ":wq" > /dev/null
```

Comment:
I added this since the most common problem I saw was trailing whitespace on both empty and nonempty lines. This makes it easy to fix those particular errors until we have our editors set up to do it automatically, at which point it's not terribly necessary.